libraries/AP_Landing/AP_Landing_Deepstall.cpp

```diff
@@ -488,7 +511,7 @@ float AP_Landing_Deepstall::update_steering()
     ab.normalize();
     Vector2f a_air = location_diff(arc_exit, current_loc);
 
-    float crosstrack_error = a_air % ab;
+    crosstrack_error = a_air % ab;
     float sine_nu1 = constrain_float(crosstrack_error / MAX(L1_period, 0.1f), -0.7071f, 0.7107f);
```

Comment:
You could have a scalar parameter, but then if you choose to retune L1 for any reason you would be detuning your deepstall which is not good. Also for example on my aircraft I use this at the moment for the tuned paths (I need to implement FBW deepstall to really aggressively test the deep stall tuning):
LAND_DS_L1      15.000000
LAND_DS_L1_I    0.000000
NAVL1_PERIOD    21.000000
NAVL1_XTRACK_I  0.020000